test/contract.errors.js

```diff
@@ -0,0 +1,166 @@
+const assert = require('assert');
+const Eth = require('../packages/web3-eth');
+const FakeIpcProvider = require('./helpers/FakeIpcProvider');
+
+const abi = [{
+    name: "simpleMethod",
+    payable: false,
+    constant: true,
+    type: "function",
+    stateMutability: "view",
+    inputs: [{ name: "a", type: "bytes32" }],
+    outputs: [{ name: "b", type: "bytes32" }],
+}];
+
+describe('contract: errors', function () {
```

Comment:
I think we should define a clear pattern to name test files and test cases. This will increase the readability of our test outputs extremely. Can you work something out as QA officer ;-P

Reply:
Any idea here @cgewecke?

Reply:
We should at least create an issue out of it.